src/pages/experiments/[experimentId]/data-processing/configure-embedding/components/EmbeddingPreview.jsx

```diff
@@ -145,14 +178,6 @@ const EmbeddingPreview = () => {
       );
     }
 
-    if (!config || (embeddingMethod && embedding[embeddingMethod]?.loading)) {
-      return (
-        <center>
-          <Spin size='large' />
-        </center>
-      );
-    }
```

Comment:
we need the spinner back

Reply:
Brought back in new commit

Reply:
The spinner is moved into the refactored component, so by default the refactored component shows spinner when its data is loading. This is done because the component has async calls inside it, so I figured it's easier to handle loading state inside the component.